scripts/approve_embargo_terminations.py

```diff
@@ -0,0 +1,70 @@
+"""EmbargoTerminationApprovals are the Sanction subclass that allows users
+to make Embargoes public before the official end date. Like RegistrationAprpovals
+and Embargoes, if an admin fails to approve or reject this request within 48
+hours it is approved automagically.
+
+
+Run nightly, this script will approve any embargo termination
+requests for which not all admins have responded within the 48 hour window.
+Makes the Embargoed Node and its components public.
+"""
+
+import datetime
+import logging
+import sys
+
+from modularodm import Q
+
+from framework.transactions.context import TokuTransaction
+from framework.celery_tasks import app as celery_app
+
+from website import models, settings
+from website.app import init_app
+
+from scripts import utils as scripts_utils
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+def get_pending_embargo_termination_requests():
+    auto_approve_time = datetime.datetime.now() - settings.EMBARGO_TERMINATION_PENDING_TIME
+
+    return models.EmbargoTerminationApproval.find(
+        Q('initiation_date', 'lt', auto_approve_time) &
+        Q('state', 'eq', models.EmbargoTerminationApproval.UNAPPROVED)
+    )
+
+def main():
+    pending_embargo_termination_requests = get_pending_embargo_termination_requests()
+    count = 0
+    for request in pending_embargo_termination_requests:
+        registration = models.Node.find_one(Q('embargo_termination_approval', 'eq', request))
+        if not registration.is_embargoed:
+            raise RuntimeError("Registration {0} associated with this embargo termination request ({0}) is not embargoed.".format(
```

Comment:
This will terminate the script early and may cause other (valid) termination requests to not be approved. It might make more sense to log the error and `continue`.